apps/rush/src/actions/RequireAction.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as colors from 'colors';
+import * as fsx from 'fs-extra';
+import * as glob from 'glob';
+import * as os from 'os';
+import * as path from 'path';
+import builtinPackageNames = require('builtins');
+import { CommandLineAction } from '@microsoft/ts-command-line';
+
+import RushCommandLineParser from './RushCommandLineParser';
+
+export default class RequireAction extends CommandLineAction {
+  private _parser: RushCommandLineParser;
+
+  constructor(parser: RushCommandLineParser) {
+    super({
+      actionVerb: 'require',
```

Comment:
What about "check-imports" ????

---
In reply to: [116320505](https://github.com/Microsoft/web-build-tools/pull/183#discussion_r116320505) [](ancestors = 116320505)